tst/RtcpFunctionalityTest.cpp

```diff
@@ -295,37 +295,75 @@ TEST_F(RtcpFunctionalityTest, onpli)
     freePeerConnection(&pRtcPeerConnection);
 }
 
+static void testBwHandler(UINT64 customData, UINT32 txBytes, UINT32 rxBytes, UINT32 txPacketsCnt, UINT32 rxPacketsCnt,
+                                                   UINT64 duration) {
+    UNUSED_PARAM(customData);
+    UNUSED_PARAM(txBytes);
+    UNUSED_PARAM(rxBytes);
+    UNUSED_PARAM(txPacketsCnt);
+    UNUSED_PARAM(rxPacketsCnt);
+    UNUSED_PARAM(duration);
+    return;
+}
+
 static void parseTwcc(const std::string& hex, const uint32_t expectedReceived, const uint32_t expectedNotReceived)
 {
+    PRtcPeerConnection pRtcPeerConnection = nullptr;
+    PKvsPeerConnection pKvsPeerConnection;
     BYTE payload[256] = {0};
     UINT32 payloadLen = 256;
     hexDecode(const_cast<PCHAR>(hex.data()), hex.size(), payload, &payloadLen);
     RtcpPacket rtcpPacket{};
+    RtpPacket rtpPacket{};
+    RtcConfiguration config{};
+    UINT64 value;
+    UINT16 twsn;
+    UINT32 extpayload, received = 0, lost = 0;
+
     rtcpPacket.header.packetLength = payloadLen / 4;
     rtcpPacket.payload = payload;
     rtcpPacket.payloadLength = payloadLen;
-    TwccManager twcc{};
-
-    ASSERT_EQ(STATUS_SUCCESS, parseRtcpTwccPacket(&rtcpPacket, &twcc));
-    ASSERT_EQ(STATUS_SUCCESS, stackQueueClear(&twcc.twccPackets, FALSE));
-
-    uint32_t received = 0;
-    uint32_t lost = 0;
-    for (const auto& packet : twcc.twccPacketBySeqNum) {
-        if (packet.remoteTimeKvs == TWCC_PACKET_LOST_TIME) {
-            lost++;
-        } else if (packet.remoteTimeKvs != TWCC_PACKET_UNITIALIZED_TIME) {
-            received++;
+
+
+    EXPECT_EQ(STATUS_SUCCESS, createPeerConnection(&config, &pRtcPeerConnection));
+    pKvsPeerConnection = reinterpret_cast<PKvsPeerConnection>(pRtcPeerConnection);
+    EXPECT_EQ(STATUS_SUCCESS, peerConnectionOnSenderBandwidthEstimation(pRtcPeerConnection, 0,
+                                                                        testBwHandler));
+
+    UINT16 baseSeqNum = getUnalignedInt16BigEndian(rtcpPacket.payload + 8);
+    UINT16 pktCount = TWCC_PACKET_STATUS_COUNT(rtcpPacket.payload);
+
+    for(auto i = baseSeqNum; i < baseSeqNum + pktCount; i++) {
```

Comment:
Let's not use auto, the standard across this code base is to declare variables at the top of the function (to be C89 compatible) explicitly.  Should be `UINT16` in this case.

Reply:
Just noticed this is in the tests, which are C++, ignore.